sklearn/cross_validation.py

```diff
@@ -1549,7 +1549,7 @@ def cross_val_score(estimator, X, y=None, scoring=None, cv=None, n_jobs=1,
     >>> y = diabetes.target[:150]
     >>> lasso = linear_model.Lasso()
     >>> print(cross_val_score(lasso, X, y))  # doctest:  +ELLIPSIS
-    [ 0.33150734  0.08022311  0.03531764]
+    [ 0.33139385  0.08022927  0.03534988]
```

Comment:
Rather than changing the numbers here, you should probably add ellipsis, so that the test is robust.